etl/steps/data/garden/sipri/2024-07-08/military_expenditure.meta.yml

```diff
@@ -0,0 +1,137 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Military Personnel & Spending
+
+  description_short_includes: Includes military and civil personnel, operation and maintenance, procurement, military research and development, infrastructure, and aid. Civil defense and current expenditures for previous military activities are excluded.
```

Comment:
I'd move the "Civil defense and current expenditures for previous
military activities are excluded." to the footnote to shorten the subtitle a bit and also so it's consistent with the other chart <img width="677" alt="Screenshot 2024-07-30 at 11 01 52" src="https://github.com/user-attachments/assets/f6fd7378-080b-42b1-bd43-af13a9c0777b">